src/modules/Lessons/8_LessonUserData/components/UserForm.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import TextInput from '../../../common/components/TextInput';
+// import RadioButtonField from './RadioButtonField';
+import RadioButton from '../../../common/components/RadioButton';
+import CheckBox from '../../../common/components/CheckBox';
+// import CheckBoxField from './CheckBoxField';
+import Dropdown from '../../../common/components/Dropdown';
+import Saving from './Saving';
+import Countries from '../../../../util/countries.js'
+
+class UserForm extends Component{
+	constructor(props){
+		super(props);
+
+		this.state = {
+			name: '',
+			gender: '',
+			education: [],
+			country: Countries[0].label,
+			isSaving: false,
+			radioButtonOptions: [{key:'male', value: 'Male'}, 
+			{key:'female', value: 'Female'}],
+			checkBoxOptions: [{key:'grad', value: 'Graduation'}, 
+			{key:'postgrad', value: 'Post Graduate'}],
```

Comment:
the state can be improved such that you can provide default values of radio buttons or dropdowns by moving them in separate JS files